docs/griptape-tools/custom-tools/index.md

```diff
@@ -4,24 +4,11 @@ Building your own tools is easy with Griptape!
 
 To start, create a directory for your tool inside your project. All tool directories should have the following components:
 
-* `manifest.yml` with a YAML manifest.
 * `tool.py` file with a tool Python class.
 * `requirements.txt` file with tool Python dependencies.
```

Comment:
@william-price01 while we're already in here, can you test if this is true? I think this is optional I could be wrong.

Reply:
Test if requirements.txt is required, if your tool must be called tool.py? 
@collindutter

Reply:
Actually, both would be good to test.